spec/schema.json

```diff
@@ -332,6 +359,50 @@
           "type": "integer"
         }
       }
+    },
+    "ActiveOfficerDetailsResponse": {
+      "type": "object",
+      "properties": {
+        "foreName1":  {
```

Comment:
the json response should be snake_case.
if the oracle api is returning camelCase.
Then you could just have 2 classes in this api.
And map from camelCase to snake_case just before returning the response.

Reply:
I think it might be worth fixing the json object formatting on the oracle-query-api side as well, rather than work around it with a 2nd class in this api.